crates/turborepo-cache/src/async_cache.rs

```diff
@@ -64,12 +64,14 @@ impl AsyncCache {
                         let permit = semaphore.clone().acquire_owned().await.unwrap();
                         let real_cache = real_cache.clone();
                         workers.push(tokio::spawn(async move {
-                            let _ = real_cache.put(&anchor, &key, &files, duration).await;
+                            if let Err(err) = real_cache.put(&anchor, &key, &files, duration).await
+                            {
+                                warn!("{err}");
```

Comment:
I'm not actually sure we want this. If the failures are correlated, it's likely that you'll get a flood of them. Maybe we want a limiter? after 5 or so, stop printing them?